spread/google.go

```diff
@@ -154,6 +154,9 @@ const googleStartupScript = `
 echo root:%s | chpasswd
 
 sed -i 's/^\s*#\?\s*\(PermitRootLogin\|PasswordAuthentication\)\>.*/\1 yes/' /etc/ssh/sshd_config
+sed -i 's/^PermitRootLogin=/#PermitRootLogin=/g' /etc/ssh/sshd_config.d/* || true
+sed -i 's/^PasswordAuthentication=/#PasswordAuthentication=/g' /etc/ssh/sshd_config.d/* || true
+test -d /etc/ssh/sshd_config.d && echo -e 'PermitRootLogin=yes\nPasswordAuthentication=yes' > /etc/ssh/sshd_config.d/00-spread.conf
```

Comment:
According to the [`sshd_config(5)`](https://manpages.ubuntu.com/manpages/jammy/en/man5/sshd_config.5.html) manpage, the config file usually doesn't have a `=` between the key/value pairs. For example:

```
% grep = /etc/ssh/sshd_config
# This sshd was compiled with PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games
```